src/public/index.html

```diff
@@ -6,6 +6,7 @@
         <script src="https://cdn.socket.io/socket.io-1.4.5.js"></script>
         <script src="https://code.jquery.com/jquery-1.11.1.js"></script>
         <script src="./js/rs.js"></script>
+        <script src="./js/util.js"></script>
```

Comment:
Ah, ok! I figured putting it before main would be enough since none of it is executed till main, but that makes sense since RS wont know what UTIL is when it is getting built at start time.